apps/learninglocker/templates/app/_env.yml.j2

```diff
@@ -0,0 +1,8 @@
+API_PORT: "{{ learninglocker_api_port }}"
+UI_PORT: "{{ learninglocker_ui_port }}"
+LOG_MIN_LEVEL: info
+LOG_DIR: /tmp
+NODE_ENV: production
+WINSTON_CLOUDWATCH_ENABLED: false
```

Comment:
What does this variable stand for?

Reply:
to enable loggin on cloudwatch. Should I put a link to all available environment variables ?

Reply:
Maybe yeah.